README.md

```diff
@@ -2,154 +2,162 @@
+
+## Quick Start
+
+### Prerequisites
+
+- **Node.js 24+** for CLI and library usage
+- **Filecoin Calibration testnet wallet** with:
+  - Test FIL for transaction gas ([Faucet](https://faucet.calibnet.chainsafe-fil.io/funds.html))
+  - Test USDFC stablecoin for storage payments ([USDFC Faucet](https://forest-explorer.chainsafe.dev/faucet/calibnet_usdfc))
+
+### Installation
+
+```bash
+npm install -g filecoin-pin
+```
+
+### Basic Usage
+
+```bash
+# 1. Configure payment permissions (one-time setup)
+filecoin-pin payments setup --auto
+
+# 2. Upload a file to Filecoin
+filecoin-pin add myfile.txt
+
+# 3. Verify storage with cryptographic proofs
+filecoin-pin data-set <dataset-id>
+```
+
+For detailed guides, see:
+- **CLI**: [Complete CLI walkthrough](https://docs.filecoin.io/builder-cookbook/filecoin-pin/filecoin-pin-cli)
+- **GitHub Action**: [CI/CD integration guide](https://docs.filecoin.io/builder-cookbook/filecoin-pin/github-action)
+- **dApp Demo**: [Browser-based implementation](https://docs.filecoin.io/builder-cookbook/filecoin-pin/dapp-demo)
 
 ## Configuration
 
 ### Environment Variables
 
 ```bash
-# Required for daemon
-PRIVATE_KEY=0x...              # Ethereum private key with USDFC
+# Required
+PRIVATE_KEY=0x...              # Ethereum private key with USDFC tokens
 
 # Optional
-RPC_URL=wss://...              # Filecoin RPC (default: calibration websocket)
-PORT=3456                      # Daemon port
-DATABASE_PATH=./pins.db        # SQLite database
-CAR_STORAGE_PATH=./cars        # CAR file directory
-LOG_LEVEL=info                 # Logging level
+RPC_URL=wss://...              # Filecoin RPC endpoint (default: Calibration testnet)
+
+# Optional for Pinning Server Daemon
+PORT=3456                      # Daemon server port
+HOST=localhost                 # Daemon server host
+DATABASE_PATH=./pins.db        # SQLite database location
+CAR_STORAGE_PATH=./cars        # CAR file storage directory
+LOG_LEVEL=info                 # Logging verbosity (info, debug, error)
 ```
 
-### Default Directories
+### Default Data Directories
 
-When not specified, data is stored in:
+When `DATABASE_PATH` and `CAR_STORAGE_PATH` are not specified, data is stored in platform-specific locations:
 - **Linux**: `~/.local/share/filecoin-pin/`
 - **macOS**: `~/Library/Application Support/filecoin-pin/`
 - **Windows**: `%APPDATA%/filecoin-pin/`
 
-## How It Works
-
-### IPFS Pinning Flow
-1. Receive pin request from IPFS
-2. Create CAR file with root CID
-3. Fetch blocks via Bitswap
-4. Stream blocks directly to CAR
-5. Upload CAR to Synapse
-6. Return Filecoin piece CID
-
-### CAR Import Flow
-1. Validate CAR file format
-2. Extract root CIDs
-3. Upload to Synapse
-4. Display storage provider info
-
 ## Development
 
+Want to contribute to Filecoin Pin or run it locally?
+
 ```bash
 # Clone and install
 git clone https://github.com/filecoin-project/filecoin-pin
```

Comment:
this block could do with a minor cleanup, it's focused on the daemon - "Run the development server" is "Run the Pinning Server"

"Build for production" could just be "Compile TypeScript source"

you could add in `node ./dist/cli.js` after `npm run build` has been done which is the equivalent of running `filecoin-pin` if you had it installed globally (which you can do with `npm install filecoin-pin -g`, I don't think we document that anywhere, it's like doing `npx filecoin-pin` but you install a `filecoin-pin` executable, but it's stuck on that version until you run `npm install filecoin-pin -g` again).

In dev, that's how I'm testing it all the time, `node ./dist/cli.js add <file>` etc.

Reply:
Good call.  I added some more comments.  More improvements to be made in the future.